docs/rendering_system.rst

```diff
@@ -0,0 +1,241 @@
+..
+   Author: Adam Turner
+
+   We can't use :pep:`X` references in this document, as they are relative from
+   the content root.
+   We use :doc:`PEP X <../pep-xxxx>` instead.
+
+
+An Overview of the PEP Rendering System
+=======================================
+
+This document provides an overview of the PEP rendering system, as a companion
+to :doc:`PEP 676 <../pep-0676>`.
+
+
+1. Configuration
+----------------
+
+Configuration is held in three files:
+
+- ``conf.py`` contains the majority of the configuration
+- ``contents.rst`` contains the table of contents directives for Sphinx
+- ``pep_sphinx_extensions/pep_theme/theme.conf`` sets the Pygments themes
+
+The configuration:
+
+- registers the custom Sphinx extension
+- sets both ``.txt`` and ``.rst`` suffixes to be parsed as PEPs
+- tells Sphinx which source files to use
+- registers the PEP theme, maths renderer, and template
+- disables some default settings that are covered in the extension
+- sets the default and 'dark mode' code formatter styles
```

Comment:
```suggestion
- sets the default and "dark mode" code formatter styles
```

Maybe BrE is different (ignore if so), but at least in AmE style guides, `"` is the first level of quote nesting, while `'` is second level

Reply:
You are of course correct. Normally writing in LaTeX forces me to think about this, but I got sloppy. 

If it's of any interest the style manual I most often use is The Economist's -- might held be in your local library's collection (although I apprecaite an esoteric text for an American library), but an older edition (9th, 2005) can be found online at http://www.benjaminjameswaddell.com/wp-content/uploads/2018/08/the-economist-style-guide.pdf

A

Reply:
Hmmm, interesting—I didn't realize they had one. Over here, AP style is the standard for press writing (aside from NYT's quaint and idiosyncratic style) and is nominally what I follow as a fallback to my Spyder documentation style guide (and those I've sourced it from) for general English, since it is what I'm used to from writing and copyediting the science mags I used to work for on the side, and is probably the most common of the established style guides on the web.

Reply:
The [Guardian style guide](https://www.theguardian.com/guardian-observer-style-guide-a) is also well established in the UK, I'm just pretentious..! (I am a student of economics, so clearly I picked the right newspaper's styles!).

I must actually read an edition of the NYT at some point, even more so if it is quaint and idiosyncratic!

A